### PR TITLE
Add autofix support for UnnecessaryReturnKeyword rule

### PR DESCRIPTION
**Pull request by jonnybot0**
## Proposed Changes

This adds support for auto-fixing the `UnnecessaryReturnKeyword` rule automatically. I've tested it locally in a repository that I was applying this rule to, and it seems to work as expected.

It's unclear to me if I need to do any additional work to document this or if additional tests would be appropriate. Likewise, the files in `lib` changed as a result of this when I ran the build. I'm not sure if I'm meant to commit those, or if they are covered by the release process.

In short, repo newbie here! Feedback welcome. :)

### CODE REVIEW

**Comment by nvuillam** [comment]
@jonnybot0 i'm sorry i critically lack of time to maintain npm-groovy-lint, and before merging other PRs we would need to succeed to upgrade to latest groovy / CodeNarc versions, but I definitely keep this PR open for the day we can merge it ! :)
